README.md

```diff
@@ -25,6 +25,18 @@ Make sure you have GCC-4.6, G++-4.6, and GFortran-4.6 installed. For
 a full list of external dependencies on Ubuntu you can look at our
 [Travis configuration](https://github.com/graalvm/sulong/blob/master/.travis.yml).
 
+On the Mac you can use Homebrew:
+
+    brew install gcc
```

Comment:
Dragonegg supports GCC versions [up to 4.8](http://dragonegg.llvm.org/). Although you get the right Fortran library with GCC 5.3, it would still be better to install a version that works with Dragonegg.